server/src/main/java/com/linecorp/centraldogma/server/internal/api/auth/RequiresRoleDecorator.java

```diff
@@ -56,13 +73,14 @@ public final HttpResponse serve(Service<HttpRequest, HttpResponse> delegate, Ser
                 if (cause != null) {
                     return handleException(cause);
                 }
-                if (!isAccessAllowed(ctx, req, user, role)) {
+                if (!user.isAdmin() && !accessibleRoles.contains(role)) {
                     return HttpApiUtil.throwResponse(
                             HttpStatus.FORBIDDEN,
-                            "You must be %s of project '%s'.", role, projectName);
+                            "You must be granted one of the following roles in project '%s': %s",
```

Comment:
in the project '%s'?
How about `You must have one of the following roles in the project '%s' to access: %s`?

Reply:
How about: `You must have one of the following roles to access the project '%s': %s`

Reply:
Yeah, it's better :)